src/views/Profile/PersonalInfo/PersonalInfo.tsx

```diff
@@ -200,6 +200,19 @@ export const PersonalInfo: React.FC<IEditAuthorProps> = ({ author }) => {
     [errorMessages],
   );
 
+  useEffect(() => {
+    const handleBeforeUnload = (e: BeforeUnloadEvent) => {
+      e.preventDefault();
+      e.returnValue =
+        'Зміни не збережені. Ви впевнені, що хочете залишити сторінку?';
```

Comment:
for old browsers which support this, right?

Reply:
Yes